container/images/run-domain-demo.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#build the images
+podman build -t localhost/jboss-eap8-domain-openjdk21-openshift ./eap8-domain-openjdk21-openshift
+podman build -t localhost/domain-controller \
+./domain-controller-image
+podman build -t localhost/host-controller \
+./host-controller-image
+
+# Create a shared network so all the machines can communicate directly using the host names
+podman network create domain-mode-network
+
+# Run the domain-controller container
+podman run -d --rm \
+  --name domain-controller \
+  --hostname primaryhostname \
+  --network domain-mode-network \
+  -p 9990:9990 \
+  -e JBOSS_EAP_DOMAIN_HOST_NAME=primary \
+  -e JBOSS_EAP_DOMAIN_USER=secondary \
+  -e JBOSS_EAP_DOMAIN_PASSWORD=secondary \
+  -e JBOSS_EAP_DOMAIN_DOMAIN_CONFIG=openshift-domain.xml \
+  -e JBOSS_EAP_DOMAIN_HOST_CONFIG=my-host-primary.xml \
+  -e SERVER_PUBLIC_BIND_ADDRESS=0.0.0.0 \
+  -e SERVER_MANAGEMENT_BIND_ADDRESS=0.0.0.0 \
```

Comment:
0.0.0.0 is the default value. Why should it be set?

Reply:
It is just for demonstrating purposes and to show all the available options, it can be removed